tools/bundle_scss/scss.cjs

```diff
@@ -0,0 +1,38 @@
+const sass = require("sass");
+const { globSync } = require("glob");
+const fs = require("fs");
+
+const { writeToFile } = require("../utils/files.cjs");
+const { bundle } = require("../bundle.cjs");
+
+function createScssBundle(target, cssOnly) {
+  const start = Date.now();
+  if (!target) {
+    throw new Error("Missing target!!");
+  }
+  const files = globSync("src/components/**/*.scss");
+  const banner = bundle.cssBanner();
+  // Build a fake index.scss file that imports all the components
+  // scss files
+  const index = files.map((file) => `@use "${file}";`).join("\n");
+  const { css } = sass.compileString(index, {
```

Comment:
Instead of building a fake index file importing all others, I think we should build the `o_spreadsheet.scss` bundle file and then compile it to css.
That way, we guarantee both are equivalent.

Everytime we use `@use`, the `css` is valid, but the scss isn't-and it's silent.
And I actually think we'll want `@use` to import variables ?

It makes me reconsider using a lib that resolves all imports. (or not providing a `scss` bundle ?)

Reply:
possible solutions are:
- no `@use`
- give css bundle to odoo
- declare all variables in a single file that we add as the first file in the bundle
- resolve `@use` ourselves
- give all scss files to odoo
   - all moved/flatten to a single directory (and rewrite `@use`
   - reproduce file hierarchy

Reply:
option 1 seems to be the easiest short term. But in the long run, we'll want the scss in odoo too, to benefit from variable overwrites

Reply:
For the record:
Option 1: Works but we cannot override variables
Option 2: Works, but we will not allow to use `@use`, it will be much easier.
Option 3: odoo does not support `@use` and `@forward`, and forbid the use of `@import`, so not a possible solution